test/core/security/grpc_audit_logging_test.cc

```diff
@@ -81,26 +82,82 @@ class AuditLoggingTest : public ::testing::Test {
 
 }  // namespace
 
-TEST_F(AuditLoggingTest, SuccessfulLoggerCreation) {
+//
+// AuditLoggerRegistryTest
+//
+
+TEST_F(AuditLoggerRegistryTest, SuccessfulLoggerCreation) {
   auto result = AuditLoggerRegistry::ParseConfig(kName, Json());
   ASSERT_TRUE(result.ok());
   ASSERT_NE(AuditLoggerRegistry::CreateAuditLogger(std::move(result.value())),
             nullptr);
 }
 
-TEST_F(AuditLoggingTest, UnknownLogger) {
+TEST_F(AuditLoggerRegistryTest, UnknownLogger) {
   auto result = AuditLoggerRegistry::ParseConfig("unknown_logger", Json());
   EXPECT_EQ(result.status().code(), absl::StatusCode::kNotFound);
   EXPECT_EQ(result.status().message(),
             "audit logger factory for unknown_logger does not exist")
       << result.status();
 }
 
-TEST_F(AuditLoggingTest, AuditLoggerFactoryExistenceChecks) {
+TEST_F(AuditLoggerRegistryTest, LoggerFactoryExistenceChecks) {
   EXPECT_TRUE(AuditLoggerRegistry::FactoryExists(kName));
   EXPECT_FALSE(AuditLoggerRegistry::FactoryExists("unknown_logger"));
 }
 
+//
+//  StdoutLoggerTest
+//
+
+TEST(StdoutLoggerTest, LoggerFactoryExistenceChecks) {
+  EXPECT_TRUE(AuditLoggerRegistry::FactoryExists("stdout_logger"));
+}
+
+TEST(StdoutLoggerTest, BadLoggerConfig) {
+  auto result =
+      AuditLoggerRegistry::ParseConfig("stdout_logger", Json::FromBool(false));
```

Comment:
I think there's some value in making this more explicit
```suggestion
auto invalid_value = Json::FromBool(false)
auto result = AuditLoggerRegistry::ParseConfig("stdout_logger", invalid_value);
```

Reply:
Sure. Done.